vendors/nordic/boards/nrf52840-dk/ports/common_io/iot_test_common_io_internal.c

```diff
@@ -29,6 +29,18 @@
 
 #include "iot_test_common_io_internal.h"
 
+#if defined( IOT_TEST_COMMON_IO_FLASH_SUPPORTED ) && ( IOT_TEST_COMMON_IO_FLASH_SUPPORTED >= 1 )
+/* FLASH */
+    void SET_TEST_IOT_FLASH_CONFIG( int testSet )
+    {
+        /* This value depends on linked flash usage. At time of writing, test project ended @ 0x49bc3
+         * Some tests require this to be page-aligned
+         */
+        ultestIotFlashStartOffset = 0x75000;
+    }
+
+#endif /* if defined( IOT_TEST_COMMON_IO_SPI_SUPPORTED ) && ( IOT_TEST_COMMON_IO_SPI_SUPPORTED >= 1 ) */
```

Comment:
Copy paste error

Reply:
Thanks. Addressed these comments in latest commit. 

Decided to remove these end-guard comments as their related code blocks were small, so these comments were just redundant and unecessary visual clutter.